index.html

```diff
@@ -43,6 +36,174 @@ <h1 class="h1-green">
       <i class="fab fa-github"></i>
       <i class="fab fa-angellist"></i>
     </nav>
+    <nav>
+      <img src="images/Action.svg" alt="Action icon">
+    </nav>
   </header>
+  <section class="works-container">
+    <header class="works-header">
+      <h1 class="h1-blue">
+        My Recent<br> Works
+      </h1>
+      <img src="images/Indicator.svg" alt="Works title Indicator">  
+    </header>
+    <section class="cards-container">
+      <article class="cards">
+        <section class="card-space"></section>
+        <section class="card-info">
+          <h4>Multi-Post Stories<br>Gain+Glory</h4>
+          <nav class="made-with">
+            <ul class="ul-with">
+              <li class="blue-li">Ruby on rails</li>
+              <li class="blue-li">css</li>
+              <li class="blue-li">JavaScript</li>
+              <li class="blue-li">html</li>
+            </ul>
+          </nav>
```

Comment:
- [ ] [Required] The use of the `nav` element here is not necessary. We can just have only the `ul` element and have the same result. **please fix this in all parts of your document where you used the `nav` to wrap the `ul` element.